### PR TITLE
Explain 2FA for back end users

### PR DESCRIPTION
**Pull request by richardhj**
The config to enforce 2FA changed since it was explained here: https://github.com/contao/core-bundle/pull/1545#issue-193781960

### CODE REVIEW

File: docs/manual/user-management/users.de.md [hunk 1]
```diff
@@ -221,6 +221,23 @@ damit die Verzeichnisstruktur schneller lädt.
 **Passwort:** Hier kannst du dem Benutzer ein Passwort zuweisen.
 
 
+### Zwei-Faktor-Authentisierung
+
+Benutzer können die Zwei-Faktor-Authentisierung aktivieren, um den Account zusätzlich abzusichern. Neben dem Benutzernamen 
+und Passwort muss ein Verifizierungscode ("Time-based One-time Password") eingegeben werden. Dieses Einmalpasswort muss von 
+einer Zwei-Faktor-App wie z.B. 1Password, Authy, Google Authenticator, Microsoft Authenticator, LastPass Authenticator oder
```
Comment (fritzmg) [inline docs/manual/user-management/users.de.md]:
```suggestion
einer Zwei-Faktor-App wie z. B. 1Password, Authy, Google Authenticator, Microsoft Authenticator, LastPass Authenticator oder 
```

**Review by fritzmg — CHANGES_REQUESTED** [review]
Can you also provide the English translation? If not, I'll add it.

**Comment by richardhj** [comment]
> Can you also provide the English translation? If not, I'll add it.

Will do

**Comment by richardhj** [comment]
Where is the English document?

**Comment by fritzmg** [comment]
> Where is the English document?

Ah sorry, actually you don't have to, because the whole English translation for this page does not exist yet ;)

**Review by fritzmg — DISMISSED** [review]

File: docs/manual/user-management/users.de.md [hunk 2]
```diff
@@ -221,6 +221,23 @@ damit die Verzeichnisstruktur schneller lädt.
 **Passwort:** Hier kannst du dem Benutzer ein Passwort zuweisen.
 
 
+### Zwei-Faktor-Authentisierung
+
+Benutzer können die Zwei-Faktor-Authentisierung aktivieren, um den Account zusätzlich abzusichern. Neben dem Benutzernamen 
+und Passwort muss ein Verifizierungscode ("Time-based One-time Password") eingegeben werden. Dieses Einmalpasswort muss
```
Comment (xchs) [inline docs/manual/user-management/users.de.md]:
```suggestion
und Passwort muss ein Verifizierungscode (»Time-based One-time Password«) eingegeben werden. Dieses Einmalpasswort muss 
```

**Review by xchs — CHANGES_REQUESTED** [review]

**Review by netzarbeiter — APPROVED** [review]
Thank you @richardhj

**Pull request merged**